frontend/pages/shopping-lists/[id].vue

```diff
@@ -154,27 +179,15 @@
         </BaseButton>
       </div>
 
-      <div
-        v-for="(value, key) in itemsByLabel"
-        :key="key"
-        class="pb-4"
-      >
-        <v-btn
-          :color="getLabelColor(value[0]) ? getLabelColor(value[0]) : '#959595'"
-          :style="{
-            'color': getTextColor(getLabelColor(value[0])),
-            'letter-spacing': 'normal',
-          }"
-          @click="toggleShowLabel(key.toString())"
-        >
-          <v-icon>
-            {{ labelOpenState[key] ? $globals.icons.chevronDown : $globals.icons.chevronRight }}
-          </v-icon>
-          {{ key }}
-        </v-btn>
-        <v-divider />
-        <v-expand-transition>
-          <div v-if="labelOpenState[key]">
+      <v-expansion-panels v-model="openedSections" multiple>
+        <v-expansion-panel v-for="(value, key) in itemsByLabel" :key="key">
+          <v-expansion-panel-title
+            :color="getLabelColor(value[0]) ? getLabelColor(value[0]) + '40' : '#ffffff40'"
```

Comment:
Some of our colors already supply the opacity (it's supported in the color picker) so this can throw an error. Let's just leave this part off.
```suggestion
            :color="getLabelColor(value[0]) ? getLabelColor(value[0]) : '#ffffff40'"
```

Reply:
Also, can we modify this to just return the default in `getLabelColor`? That way we can get rid of this ternary.

Reply:
I opted to just have remove the ternary and not set a color if one isn't present. 

I think this works as it gives us a more theme-neutral default. (i.e., black in dark mode and white in light mode). That way if someone doesn't use labels, the section blends into the ui a bit more. 

If you feel strongly about adding the default color in the `getLabelColor` I can make the change but I figure I'd provide this option first.

Reply:
Ah perfect, I like that better